app/toolbar/components/status/states/Confirm.jsx

```diff
@@ -82,26 +100,38 @@ class Confirm extends React.Component<Props> {
     handleds confirm action buttons
   */
   confirmAction = confirm => {
-    const { props } = this;
-    const { category } = props;
+    const { props, state } = this;
+    const { category, storage } = props;
     // TODO check config to see if setting is remembered
-    const validateDockerClose = true;
+    const shouldCloseDockerConfig = storage.get('closeDockerConfirm');
+    const validateDockerClose = shouldCloseDockerConfig === undefined;
 
     if (category === 'closeDocker') {
       this.stopGigantumState();
       this.handleGigantumClose(confirm);
+      if (state.isChecked) {
+        storage.set(`${category}Confirm`, confirm);
+      }
     } else if (category === 'closeGigantum') {
       if (!confirm) {
         props.transition(CANCEL, {
           message: 'Click to Quit'
         });
       } else if (validateDockerClose) {
-        props.transition(REPROMPT, {
-          message: 'Would you like to close Docker?',
-          category: 'closeDocker'
-        });
+        this.setState(
+          {
+            isChecked: false
+          },
+          () => {
+            console.log('this ran');
+            console.log(this.state);
+            props.transition(REPROMPT, {
+              message: 'Would you like to close Docker?',
+              category: 'closeDocker'
```

Comment:
i think we should use the convention for stings to be `${action}.${object}` eg 'close.docker'
```suggestion
              category: 'close.docker'
```